packages/react-server/src/ReactFlightServer.js

```diff
@@ -968,6 +968,7 @@ function callWithDebugContextInDEV<A, T>(
   // a fake owner during this callback so we can get the stack trace from it.
   // This also gets sent to the client as the owner for the replaying log.
   const componentDebugInfo: ReactComponentInfo = {
+    name: null,
```

Comment:
Found a bug with the previous isReactComponentInfo check and this hack. Since this doesn't have a name of a component is wasn't being detected. Since this is not outlined it also goes through the renderConsoleValue path.